src/Modules/CalcPerform.lua

```diff
@@ -387,11 +387,13 @@ local function doActorLifeMana(actor)
 	local breakdown = actor.breakdown
 	local condList = modDB.conditions
 
-	-- This is hacky, but currently the only way to bring data into ConfigOptions for dynamically updated tooltips
 	local lowLifePerc = modDB:Sum("BASE", nil, "LowLifePercentage")
-	data.misc.configurable.LowLifePercentage = 100.0 * (lowLifePerc > 0 and lowLifePerc or data.misc.LowPoolThreshold)
+	output.LowLifePercentage = 100.0 * (lowLifePerc > 0 and lowLifePerc or data.misc.LowPoolThreshold)
 	local fullLifePerc = modDB:Sum("BASE", nil, "FullLifePercentage")
-	data.misc.configurable.FullLifePercentage = 100.0 * (fullLifePerc > 0 and fullLifePerc or 1.0)
+	output.FullLifePercentage = 100.0 * (fullLifePerc > 0 and fullLifePerc or 1.0)
+	-- This is hacky, but currently the only way to bring data into ConfigOptions for dynamically updated tooltips
```

Comment:
You can actually pass self.build to tooltipFunc call in ConfigTab and then use tooltip = LowLifeTooltip in ConfigOptions and then you can reference this via `build.calcsTab.mainOutput.LowLifePercentage`, I think that should work. And then you can get rid of this workaround